humble/distribution.yaml

```diff
@@ -7039,6 +7039,20 @@ repositories:
       url: https://github.com/ros/ros_tutorials.git
       version: humble
     status: maintained
+  tuw_msgs:
+    release:
```

Comment:
Would you like to add a `doc` and a `source` section?  That was done for ROS 1 releases, e.g. https://github.com/ros/rosdistro/blob/57b8bfd2d425aeba494a33b3834af10e9dbefc21/noetic/distribution.yaml#L11135-L11144